src/main/java/xbot/common/advantage/AKitLogger.java

```diff
@@ -16,7 +16,7 @@ public class AKitLogger {
     public enum LogLevel {
         DEBUG, INFO
     }
-    private static LogLevel globalLogLevel = LogLevel.DEBUG;
+    private static LogLevel globalLogLevel = LogLevel.INFO;
```

Comment:
Do we want this global change to be the default for now?

Reply:
Yes, I think we should get into a pattern where it's always "INFO" in the store and only temporarily "DEBUG" in local testing. Ideally it would be a deploy-time variable (but I couldn't figure out how to make that work in IntelliJ)